src/Compilers/CSharp/Test/Semantic/Semantics/InitOnlyMemberTests.cs

```diff
@@ -2172,15 +2172,18 @@ public void InitOnlyOnMembersOfRecords()
 
             var cMembers = comp.GlobalNamespace.GetMember<NamedTypeSymbol>("C").GetMembers();
             // PROTOTYPE(init-only): expecting an 'init' setter
-            AssertEx.SetEqual(new[] {
+            AssertEx.SetEqual(cMembers.ToTestDisplayStrings(), new[] {
```

Comment:
>AssertEx.SetEqual(cMembers.ToTestDisplayStrings(), [](start = 12, length = 50)

nit: The re-ordering of arguments seems wrong. The first parameter of `SetEqual` is "expected".

Reply:
Actually, this is the `params` overload that has the order swapped, but that was pretty confusing, so I added an overload without `params` that does things the right way around.